packages/suite/src/middlewares/suite/analyticsMiddleware.ts

```diff
@@ -32,7 +33,17 @@ const analytics = (api: MiddlewareAPI<Dispatch, AppState>) => (next: Dispatch) =
     const state = api.getState();
 
     switch (action.type) {
-        case STORAGE.LOADED:
+        case STORAGE.LOADED: {
+            // get OS version of the desktop app (for web returns resp:undefined)
+            let desktopOSVersion;
+            const resp = await getOSVersion();
```

Comment:
this works for web? it looks kind of misfit here. let me think about it

Reply:
For web no particular error is thrown, but `await getOSVersion()` return `undefined`. But  there might be a better way how to detect if the suite is running on desktop

Reply:
you also don't need necessarily to send this in information in suite-ready event but you can create a new analytics event and dispatch it in the end of analyticsActions.init

Reply:
Yes maybe creating a custom action that would send `desktopOSVersion` analytics only if desktop Suite is being used would be better